src/python/merge_tester.py

```diff
@@ -7,7 +7,7 @@
 
 This script takes a csv of repos and a csv of merges and performs the merges with
```

Comment:
It would be useful to state the file format of the input and output files. (A cross-reference to somewhere else that the file formats are explained would be fine, too.)